filebeat/module/haproxy/http/_meta/fields.yml

```diff
@@ -0,0 +1,148 @@
+    
+    type: long
+  - name: frontend_name
+    description: frontend_name is the name of the frontend (or listener) which received and processed the connection.
+    
+  - name: backend_name
+    description: backend_name is the name of the backend (or listener) which was selected to manage the connection to the server.
+    
+  - name: server_name
+    description: server_name is the name of the last server to which the connection was sent.
+    
+  - name: time_client_req
+    description: time_client_req is the total time in milliseconds spent waiting for a full HTTP request from the client (not counting body) after the first byte was received.
+    
+    type: long
+  - name: time_queue
+    description: time_queue is the total time in milliseconds spent waiting in the various queues.
+    
+    type: long
+  - name: time_backend_connect
+    description: time_backend_connect is the total time in milliseconds spent waiting for the connection to establish to the final server, including retries.
+    
+    type: long
+  - name: time_server_response
+    description: time_server_response is the total time in milliseconds spent waiting for the server to send a full HTTP response, not counting data.
+    
+    type: long
+  - name: time_duration
+    description: time_duration is the time the request remained active in haproxy, which is the total time in milliseconds elapsed between the first byte of the request was received and the last byte of response was sent.
+    
+    type: long
+  - name: status_code
+    description: status_code is the HTTP status code returned to the client.
+    
+    type: long
+  - name: bytes_read
+    description: bytes_read is the total number of bytes transmitted to the client when the log is emitted.
+    
+    type: long
+  - name: captured_request_cookie
+    description: >
+      captured_request_cookie is an optional "name=value" entry indicating that the client had this cookie in the request.
+    
+  - name: captured_response_cookie
+    description: >
+      captured_response_cookie is an optional "name=value" entry indicating that the server has returned a cookie with its response.
+    
+  - name: termination_state
+    description: termination_state is the condition the session was in when the session ended.
+    
+  - name: actconn
+    description: actconn is the total number of concurrent connections on the process when the session was logged.
+    
+    type: long
+  - name: feconn
+    description: feconn is the total number of concurrent connections on the frontend when the session was logged.
+    
+    type: long
+  - name: beconn
+    description: beconn is the total number of concurrent connections handled by the backend when the session was logged.
+    
+    type: long
+  - name: srvconn
+    description: srv_conn is the total number of concurrent connections still active on the server when the session was logged.
```

Comment:
These field names are a bit cryptic, maybe we could have a `connections` object with different counts, so these fields are named `connections.active`, `connections.frontend`, `connections.backend`, `connections.server`, wdyt?

Reply:
I copied the names of the fields directly from the HAProxy documentation
https://cbonte.github.io/haproxy-dconv/1.7/configuration.html#8.2.3

I don't mind changing them though.

Reply:
Would you move them up from the fileset though? e.g. `haproxy.connections.*` as opposed to `haproxy.http.connections.*`

Reply:
I think we could leave them directly on `haproxy.connections`, so if at some moment we have also a TCP fileset they can be aggregated more easily.

Reply:
Good point, I also think it would also make sense to move these fields to the module level.